Xamarin.Forms.Platform.iOS/ContextScrollViewDelegate.cs

```diff
@@ -196,24 +210,25 @@ protected override void Dispose(bool disposing)
 
 			if (disposing)
 			{
+				ClearCloserRecognizer();
```

Comment:
Since we have _`scrollView`, we can dispose its closers properly. This method also nullifies `_table`.